eslint/cactbot-output-strings.js

```diff
@@ -54,12 +57,43 @@ const ruleModule = {
       return propKeys;
     };
 
+    const arrayContainSameElement = (arr) => {
+      return arr.every((v) => JSON.stringify(v) === JSON.stringify(arr[0]));
+    };
 
     return {
-      'Program > VariableDeclaration > VariableDeclarator > ObjectExpression'(node) {
+      /**
+       *
+       * @param node {t.ObjectExpression}
+       */
+      'Property[key.name=/outputStrings/] > ObjectExpression': function(node) {
+        const outputTemplateKey = {};
+        for (const outputString of node.properties) {
+          // each outputString
+          const values = [];
+          outputString.value.properties.forEach((x) => {
```

Comment:
The `outputString` here can be `SpreadElement`, so you have to filter(and add its properties with the `getAllKeys` function) it.
For example using the `@babel/types` helper functions.